assignment-5/insertApp.js

```diff
@@ -0,0 +1,20 @@
+module.exports.insertApp = function(app, user) {
```

Comment:
@JainVayam, Need to do following changes - 
- Instead of creating the own logic for finding the index. Use the prototype function (search the function in Array prototype)
- You're not sending any response when the user has been successfully added. (bad coding practices, Please remember that you should send the status code and message success/failure in response )
- res.send('Failed') (not sending the error codes with failure message)
- Modify the variable name localIndex to 'index' or something specific.
- Instead of assigning req.body parameters one by one, why not assigning them directly. 

Despite, I had been aware you with response sending style, you didn't implement/explore this.